### ISSUE TITLE
allow removing status bar/addon bar icon

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#27]
```
imported trac ticket
created: 2009-08-06 11:28:11
reporter: justin
```

Some users would like to use only the toolbar button and not have the status bar icon visible at all. This will require being careful with the possibility that users may not have either the status bar icon or the toolbar visible. In that case, they would still have the context menu, though.

We could try to warn the user if they disable the status bar icon when they don't have the toolbar button in use, but that's not perfect because they could remove the toolbar button later. Probably best is to just warn the user when they disable the status bar icon.

Somewhat related to this is #25.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2010-05-01 20:54:28
author: nathantuggy
```

I'd really like to have this -- RP, Greasemonkey, and Firebug are the main remaining extensions that still require the status bar, which I'd like to be able to hide.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-02-06 11:00:19
author: justin
```

This is a bit more important now with Firefox 4. The user can add the toolbar icon for RP to the addons bar (which has replaced the status bar). This toolbar icon can be placed anywhere on the addon bar, but the RP status bar icon cannot be moved. So, either the user chooses to have a fixed location RP icon or they have to have both the RP toolbar icon and the RP status bar icon in the addons bar.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-03-04 13:25:00
author: justin
```

Fixed in r427. This doesn't try to warn the user if they've left themselves no obvious way to find the menu. I've changed my mind about that being important, especially as many users won't have a status bar at all as of Fx 4.

This ticket will be closed after I've opened a new one for babelzilla translations to accompany the new string introduced in this patch.
